### DISCUSSION TITLE
Uma melhora de performance do emulador no futuro?

### DISCUSSION THREAD

**Discussion by Edblue7162** [rafaelvcaetano/melonDS-android#1137] [Q&A]
Olá, sou um grande fã do trabalho que foi posto aqui nesse projeto, fiquei muito feliz em ver que isso partiu de um Brasileiro, gostaria de fazer uma pergunta quanto a performance do emulador para o futuro, atualmente eu sei que a sua situação não o deixa muito apto para trabalhar abertamente nesse projeto, há muitas demandas pessoais que lhe tomam tempo e entendo isso, estou na mesma situação com a faculdade também, mas gostaria de saber se um dia a performance em dispositivos ARMv7-A pode ser melhorada.

Não tenho muito conhecimento acerca do processo de emulação mas acho que não é tão simples quanto parece, para dispositivos ARM64 o JIT Recompiler ajuda nesse processo, tanto que o MelonDS atualmente consegue executar os jogos em uma velocidade excelente em dispositivos ARM64, mas em dispositivos ARMv7-A eu ainda não tenho ideia de como isso poderia ser possível, eu espero um dia ver esse emulador funcionando perfeitamente em dispositivos ARMv7-A, eu nunca tinha visto uma emulação tão precisa de um Nintendo DS e isso me cativou bastante nesse projeto, Obrigado pela atenção.

**Comment by Swappo91** [comment]
Maybe on 2077

**Reply by Edblue7162** [reply]
lol

**Comment by Bardock88** [comment]
> Olá, sou um grande fã do trabalho que foi posto aqui nesse projeto, fiquei muito feliz em ver que isso partiu de um Brasileiro,

Que bom que tem mais pessoas que são também fãs deste projeto como eu, mas como você pode ver o autor do projeto de porte para o Android, parece ser português, veja esta imagem:⬇
![rafaelvcaetano (Rafael Caetano)_1](https://user-images.githubusercontent.com/102046116/234590010-95ec94ec-510b-4519-a054-4a9d58a32d81.jpg)
mas nada impede também de ser um brasileiro que mora em Portugal. Só perguntando para ele para saber.
